compat/win32/wsl.c

```diff
@@ -54,6 +54,7 @@ int copy_wsl_mode_bits_from_disk(const wchar_t *wpath, ssize_t wpathlen,
 	return ret;
 }
 
+#ifndef LX_FILE_METADATA_HAS_UID
```

Comment:
Since this modifies a file that is not part of upstream Git yet, could you please mark this as a `fixup!` of 8342d6d757e6e24b8043029dba0fe87a729c8f41 instead of making it a stand-alone patch?

Reply:
Done